tests/Test.ADAL.NET.Unit/Mocks/MockHelpers.cs

```diff
@@ -113,6 +113,12 @@ public static HttpResponseMessage CreateHttpErrorResponse()
                     "{\"ErrorSubCode\":\"70323\",\"error\":\"invalid_grant\",\"error_description\":\"AADSTS70002: Error validating credentials.AADSTS70008: The provided access grant is expired or revoked.Trace ID: f7ec686c-9196-4220-a754-cd9197de44e9Correlation ID: 04bb0cae-580b-49ac-9a10-b6c3316b1eaaTimestamp: 2015-09-16 07:24:55Z\",\"error_codes\":[70002,70008],\"timestamp\":\"2015-09-16 07:24:55Z\",\"trace_id\":\"f7ec686c-9196-4220-a754-cd9197de44e9\",\"correlation_id\":\"04bb0cae-580b-49ac-9a10-b6c3316b1eaa\"}");
```

Comment:
We should not create error messages that have messages specific to certain failures. Consider creating a generic error response. there should be helper functions to do this already